borderify_all.jsx

```diff
@@ -0,0 +1,18 @@
+#include "scripts/borderify.jsx";
+
+// File path to main working directory
+var file_path = File($.fileName).parent.fsName;
+
+// Get an array of each file in the source folder
+folder = new Folder(file_path + "/out");
+files_array = folder.getFiles();
+
+// Run the script on each image in the source folder
+for (var n = 0; n < files_array.length; n++) {
+    var file = files_array[n];
+
+    // Ensure the image can be borderify'd, then do it
+    if (file.constructor != Folder && file.name != ".DS_Store") {
```

Comment:
It's been like 4 years since I last wrote a script for photoshop, but if I recall the `Folder` class supports filtering by file type which might help you here instead of doing a file name or type check.

Looking at an old script I had
```
var fileList = folder.getFiles(/.(jpg|png|tif|psd)$/i);
```

Reply:
and actually looking at that regular expression something like `/.\.(jpg|png|tif|psd)$/i` would probably be better to avoid random name conflicts.

Reply:
oh neat! i'll update the project to use that method